distributed/worker_state_machine.py

```diff
@@ -70,6 +70,8 @@
 else:
     TaskStateState = str
 
+Type = type
```

Comment:
What mypy madness makes this necessary?

Reply:
Specifically, I think mypy is confused by `type` also being an instance variable on the dataclass.

Reply:
but how comes it worked before?!?

Reply:
`mypy` only got confused in the scope of `dummy`, where it suddenly had 3 options for `type`:
* instance variable
* type class
* function parameter